Gemfile

```diff
@@ -49,6 +49,8 @@ group :development, :test do
   gem 'bixby' # bixby == the hydra community's rubocop rules
   gem 'selenium-webdriver'
   gem 'chromedriver-helper'
+  # Alternative web driver to selenium; requires phantomjs
+  gem 'poltergeist'
```

Comment:
@afred can we get rid of this, if we are not using this anymore ?

Reply:
yep, sorry, stand by